plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -275,27 +268,23 @@ SecRule REQUEST_FILENAME "@endsWith /apps/text/public/session/push" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS"
 
-# Text app content can be anything
-SecRule REQUEST_FILENAME "@endsWith /apps/text/session/push" \
-    "id:9508129,\
-    phase:1,\
-    pass,\
-    t:none,\
-    nolog,\
-    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
-    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS"
-
 # Viewing files within the trashbin
 # Content is XML so the XML parser is switched on
-SecRule REQUEST_FILENAME "@rx /remote\.php/dav/trashbin/[^/]+/trash/$" \
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/trashbin/[^/]+/trash/" \
     "id:9508170,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
-    ctl:requestBodyProcessor=XML,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
-    setvar:'tx.allowed_request_content_type=%{tx.allowed_request_content_type} |text/plain|'"
+    chain"
+    SecRule REQUEST_HEADERS:Content-Type "@rx ^text/plain(?:\;charset=UTF-8)?$" \
+        "t:none,\
+        chain"
+        SecRule REQUEST_METHOD "@streq PROPFIND" \
```

Comment:
Does this check also work for older versions?

Reply:
Yes, the FP was introduced in Nextcloud 27 Hub 5 (Previous release) and it works there.
I encountered another FP while double checking this and added another commit for it